src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -346,25 +346,20 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
   override def loadUser(userId: WorkbenchUserId): IO[Option[WorkbenchUser]] = {
     runInTransaction { implicit session =>
       val userTable = UserTable.syntax
-      val userColumn = UserTable.column
 
-      import SamTypeBinders._
-      val loadUserQuery = samsql"select ${userTable.id}, ${userTable.email}, ${userTable.googleSubjectId} from ${UserTable.table} where ${userTable.id} = ${userId}"
-      loadUserQuery.map(rs => WorkbenchUser(rs.get[WorkbenchUserId](userColumn.id), rs.stringOpt(userColumn.googleSubjectId).map(GoogleSubjectId), rs.get[WorkbenchEmail](userColumn.email)))
-        .single().apply()
+      val loadUserQuery = samsql"select ${userTable.resultAll} from ${UserTable as userTable} where ${userTable.id} = ${userId}"
```

Comment:
Using `.resultAll` here and elsewhere so that the ResultSet will have the correct scalike-generated column aliases. As an example, here's a query that is generated using the `.resultAll`: 

```sql
select SAM_USER.id as i_on_SAM_USER, SAM_USER.email as e_on_SAM_USER, SAM_USER.google_subject_id as gsi_on_SAM_USER from SAM_USER where SAM_USER.id = testUser;
```
Without the `.resultAll` and the resulting SQL it generates, `loadUserQuery.map(UserTable(userTable))` would fail because it wouldn't be able to find the column `i_on_SAM_USER` which is what Scalike is looking for when we use `UserTable.apply`.